modules/010-vertical-pod-autoscaler-crd/crds/verticalpodautoscaler.yaml

```diff
@@ -71,7 +71,7 @@ spec:
                               resources in a ResourceList.
                             type: string
                           type: array
-                          x-doc-default: ['ResourceCPU', 'ResourceMemory']
+                          x-doc-default: ['cpu', 'memory']
```

Comment:
what about adding enum parameter with the same value as it is done in the original CRD?